src/web/app/src/main/resources/geostore-spring-security.xml

```diff
@@ -39,35 +39,52 @@
 	<bean id="geoStoreUserServiceAuthenticationProvider"
 		class="it.geosolutions.geostore.services.rest.security.UserServiceAuthenticationProvider">
 	</bean>
+
+	<!-- OAuth2 beans -->
+	<context:annotation-config/>
+
+	<bean id="googleSessionServiceDelegate"
+		  class="it.geosolutions.geostore.services.rest.security.oauth2.google.GoogleSessionServiceDelegate">
+		<constructor-arg ref="restSessionService"/>
+	</bean>
+
+	<bean id="googleSecurityConfiguration" class="it.geosolutions.geostore.services.rest.security.oauth2.google.OAuthGoogleSecurityConfiguration"/>
+
+	<!-- END OAuth2 beans-->
+
 	
 	<bean class="it.geosolutions.geostore.services.rest.security.UserAttributeTokenAuthenticationFilter"
         id="authenticationTokenProcessingFilter">
 	</bean>
-	
+
 	<bean class="it.geosolutions.geostore.services.rest.security.SessionTokenAuthenticationFilter"
         id="sessionTokenProcessingFilter">
 	</bean>
-	
+
+	<bean id="sessionServiceDelegate"
+		  class="it.geosolutions.geostore.services.rest.impl.SessionServiceDelegateImpl">
+		<constructor-arg ref="restSessionService"/>
+	</bean>
+
 	<!-- bean class="it.geosolutions.geostore.services.rest.security.WebServiceTokenAuthenticationFilter"
         id="authenticationTokenProcessingFilter">
-        <constructor-arg><value>http://localhost:8080/geoserver/www/{token}.txt</value></constructor-arg>
+        <constructor-arg><value>http://localhopst:8080/geoserver/www/{token}.txt</value></constructor-arg>
     </bean-->
 
-	<!-- Inject into the Authentication Manager the GeoStore Auth Provider -->
 	<security:authentication-manager>
 		<security:authentication-provider ref='geoStoreUserServiceAuthenticationProvider' />
 		<!-- security:authentication-provider ref='geostoreLdapProvider' / -->
 	</security:authentication-manager>
 
-    <!-- SSO headers based authentication provider
```

Comment:
Same here, we need some lines to explain the commented xml. I think this line have to be moved in authentication-manager, while the other beans need their description about the header authentication.

Reply:
fixed